mflike/MFLike.yaml

```diff
@@ -113,21 +117,55 @@ params:
       max: 2.4
     proposal: 0.03
     latex: \beta_c
-  n_CIBC:
-    prior:
-      min: 1.0
-      max: 1.4
-    proposal: 0.045
-    latex: n_\mathrm{CIBC}
   a_s:
     prior:
       min: 2.8
       max: 3.4
     proposal: 0.01
     latex: a_s
+  a_g:
+    prior:
+      dist: norm
+      loc: 2.79
+      scale: 0.45
+    proposal: 0.4
+    latex: a_\mathrm{dust}^\mathrm{TT}
+  a_gte:
+    prior:
+      min: 0
```

Comment:
why there are no `max` value ? can you provide a proposal scale (even a large one)  ?